anyway/widgets/suburban_widgets/fatal_accident_yoy_same_month.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict
+
+from flask_babel import _
+
+from anyway.request_params import RequestParams
+from anyway.backend_constants import AccidentSeverity
+from anyway.models import AccidentMarkerView
+from anyway.widgets.widget import register
+from anyway.widgets.suburban_widgets.sub_urban_widget import SubUrbanWidget
+from anyway.widgets.widget_utils import (
+    get_accidents_stats,
+)
+from anyway.models import AccidentMarker
+
+
+@register
+class FatalAccidentYoYSameMonth(SubUrbanWidget):
+    name: str = "fatal_accident_yoy_same_month"
+
+    def __init__(self, request_params: RequestParams):
+        super().__init__(request_params, type(self).name)
+        self.rank = 8
+
+    def generate_items(self) -> None:
+        latest_created_date = AccidentMarker.get_latest_marker_created_date()
+        self.items = get_accidents_stats(
+            table_obj=AccidentMarkerView,
+            filters={"accident_month": latest_created_date.month, "accident_severity": AccidentSeverity.FATAL.value},
```

Comment:
One more note - I think that the data is not exactly what widget requires - since in Widget we count killed people rather than fatal accidents - so we need to query a different table here to get the right data

Reply:
Yes, you are right. I'll change the code to count killed and not fatal accidents

Reply:
done